### ISSUE TITLE
Pkg not defined

### ISSUE THREAD

**Issue by natesire** [robertfeldt/AutoTest.jl#1]
I just installed Julia with brew on osx. I receive this error when I follow the instructions.

julia> Pkg.clone("https://github.com/robertfeldt/AutoTest.jl")
ERROR: UndefVarError: Pkg not defined
Stacktrace:
 [1] top-level scope at REPL[1]:1

**Comment by robertfeldt** [comment]
Yes, sorry, this package has not been updated in a very long while. There is a new, internal version with a new design we have developed in my research group so will be updating within a few months also here. Stay tuned.

**Comment by natesire** [comment]
Thank you. Julia is an awesome language.
